src/axom/core/tests/utils_utilities.hpp

```diff
@@ -36,7 +36,7 @@ void check_random_real(int offset)
 //------------------------------------------------------------------------------
 // UNIT TESTS
 //------------------------------------------------------------------------------
-TEST(utils_Utilities, log2)
+TEST(utils_utilities, log2)
```

Comment:
These tests weren't being run because the name didn't match the file name and we were filtering them out.